### PR TITLE
adding minecard-page

### PR DESCRIPTION
**Pull request by harsh-quadB**
missed card
my-card
## Description
Please include a summary of the changes and the purpose of this PR.

- What does this PR do?
- Why is it needed?

## Screenshots
Include screenshots for **both desktop and mobile** versions.

### Desktop:
[Add screenshot here]

### Mobile:
[Add screenshot here]

## Approach
Briefly explain the approach taken to implement the changes. 

- How did you solve the problem?
- Any particular design patterns or optimizations?

## Issue Number (if any)
Link to the issue this PR addresses, if applicable:

Fixes #[issue_number]

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by harsh-quadB** [comment]
By mistake I sent you the old imageUI  here 

is the new UI image

File: src/game_frontend/src/components/dashboard/Earn/MineCardTabSwitch.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const MineCardTabSwitch: React.FC<MineCardTabSwitchProps> = ({ myCardsData, missedCardsData }) => {
+  return (
+    <div className="mine-card-container">
+      <div className="mine-card-inner">
+        <Tabs>
+          <Tab index={0}><p>My Card</p></Tab>
+          <Tab index={1}><p>Missed Card</p></Tab>
+
+          <TabPanel index={0}>
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/dashboard/Earn/MineCardTabSwitch.tsx:30]: Make this a seperate Component

File: src/game_frontend/src/components/dashboard/Earn/MineCardTabSwitch.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                  clickPerHour={card.clickPerHour}
+                  level={card.level}
+                  cost={card.cost}
+                  image={card.image}
+                />
+              ))}
+            </div>
+          </TabPanel>
+
+          <TabPanel index={1}>
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/dashboard/Earn/MineCardTabSwitch.tsx:47]: This should be a seperate component too

File: src/game_frontend/src/pages/Dashboard/MinesCardPage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import MineCardTabSwitch from '../../../components/dashboard/Earn/MineCardTabSwitch';
+import './index.css';
+
+const myCardsData = [
```
Comment (ArjunQBTech) [inline src/game_frontend/src/pages/Dashboard/MinesCardPage/index.tsx:5]: Capitalize the STATIC VARIABLE

File: src/game_frontend/src/pages/Dashboard/MinesCardPage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    subtitle: "Future of trading",
+    profitPerHour: 2.0,
+    clickPerHour: 150,
+    level: 0,
+    cost: 8,
+    image: "/assets/images/mine03.svg"
+  }
+];
+
+const missedCardsData = [
```
Comment (ArjunQBTech) [inline src/game_frontend/src/pages/Dashboard/MinesCardPage/index.tsx:68]: Capitalize the STATIC VARIABLE

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]

**Pull request merged**